main.py

```diff
@@ -0,0 +1,77 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import schemas
+from crud import (
+    get_all_authors,
+    create_author,
+    get_author_detail,
+    get_all_books,
+    create_book,
+)
+from database import SessionLocal
+
+
+app = FastAPI()
+
+API_PREFIX = "/api/v1/"
```

Comment:
It's a good practice to keep constants like `API_PREFIX` in a separate config file. This way, if you need to change the value, you don't have to search through the code. It also helps to keep the code clean and organized.